src/components/Table/ImpactUnitTable/ImpactUnitTableContainer.tsx

```diff
@@ -62,12 +62,12 @@ function ImpactUnitContainer({
 
 	const toggleDialogs = (index: number, val: boolean) => {
 		setOpenDialogs((openStatus) =>
-			openStatus.map((element: boolean, i) => (i == index ? val : element))
+			openStatus.map((element: boolean, i) => (i === index ? val : element))
```

Comment:
Have seen this particular line more than open . 
Can be extracted to a function